src/oss/langchain-quickstart.mdx

```diff
@@ -159,7 +159,7 @@ Let's walk through each step:
                 return USER_LOCATION[user_id];
             },
             {
-                name: "get_user_info",
+                name: "get_user_location",
```

Comment:
This name should be updated in the description above too (replace `getUserInfo`)

Reply:
great callout -- more consistency 💪🏻